protocol/test/dao/Bonding.test.js

```diff
@@ -1,3 +1,4 @@
+const {ZERO_ADDRESS} = require("@openzeppelin/test-helpers/src/constants");
```

Comment:
Pedantic style comment: I notice you space your braces everywhere but here.

Reply:
nice, fixed.